.build/tar/generate_tar.sh

```diff
@@ -21,8 +25,8 @@ cp bin/$OTELCOL_BINARY dist/
 
 # compress the binary and the config into a .tar file
 echo "Compressing..."
-cd dist && tar -cvzf gcp-otel.tar.gz $OTELCOL_BINARY $CONFIG_FILE
+cd dist && tar -cvzf google-cloudops-opentelemetry-collector.tar.gz $OTELCOL_BINARY $CONFIG_FILE
```

Comment:
Just keeping the name consistent with other package files.  @JingboWangGoogle Feel free to change it back if you don't think this is a representative name

Reply:
Actually, can we reuse the variable $OTELCOL_BINARY in this case?